bindata/network/multus/multus.yaml

```diff
@@ -747,6 +747,8 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: spec.nodeName
+        - name: WHEREABOUTS_NAMESPACE
+          value: "openshift-multus"
```

Comment:
Thank you! Can we also remove the above duplicated `env` section @ https://github.com/openshift/cluster-network-operator/blob/master/bindata/network/multus/multus.yaml#L729-L733 thanks

Reply:
wow, really good catch.

I guess this was overridding the `env` stanza ...